src/core/make-widget.js

```diff
@@ -30,6 +31,9 @@ const defaultOptions = {
 
 const queryStringKeys = {
   mode: 'typeform-embed',
+  source: 'typeform-source',
+  medium: 'typeform-medium',
+  mediumVersion: 'typeform-medium-version',
```

Comment:
From [the ticket](https://jira.typeform.tf/browse/DIST-401):

> mediumVersion  ... If medium is embed-sdk, set to **current embed sdk version**, otherwise set to **empty string**

I think you need to set the mediumVersion value to current embed library version. The ticket does not say it can be overridden, but I guess it's fine.

Reply:
To my understanding, the mediumVersion is the version of whatever medium we are using. E.g. Embed-skd, embed-snippet, embed-wordpress-plugin, embed-hotspot-plugin, etc. So it is not necessary always the embed-sdk library version. 

However, if we have to default to embed-sdk, because medium and/or mediumVersion are not provided, then yes, we could use the embed library version. I guess we will need to do some versioning work (e.g. how do we get the actual version number in our code?), and this should probably be implemented in another ticket.

Reply:
> if we have to default to embed-sdk, because medium and/or mediumVersion are not provided, then yes, we could use the embed library version

Thats correct. Is there a ticket for that? From ticket description my understanding is it should be part of this ticket.

But I think we can not read current lib version right now. @prilutskiy do you have any ideas on how to access it?